spec/peep_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'peep'
+
+describe Peep do
+
+  context '#create' do
```

Comment:
To distinguish between class methods and instance methods:
Class method `context '.method' do`
Instance method `context '#method' do`